common/lib/xmodule/xmodule/course_module.py

```diff
@@ -741,6 +741,31 @@ class CourseFields(object):
         scope=Scope.settings,
     )
 
+    specialization_title_en = String(
```

Comment:
Please a comment line above this that clearly says `# Edraak(progs): link course and specialization`